src/gridster.service.ts

```diff
@@ -169,10 +169,11 @@ export class GridsterService {
 
         this.removePositionHighlight();
 
+        this.gridsterComponent.isResizing = false;
```

Comment:
I'm worry about changing the order of this line, but it's necessary.
Without this change gridster lines are visible after we stop resizing widget, because `isResizing: true`